docs/source/wavefunction_simulator.rst

```diff
@@ -58,96 +58,64 @@ amplitudes directly or look at a dictionary of associated outcome probabilities.
-a superposition of Heads and Tails proportional to the quantum state |H⟩+|T⟩. Then no matter whether Picard flips the penny
-or not, it will stay in a superposition (though the relative sign might change). In the third step Q undoes the superposition
-and always finds the penny to shows Heads.
-
-Let's see how this works!
-
-To simulate the game, we first construct the corresponding quantum circuit, which takes two qubits – one to simulate
-Picard’s choice whether or not to flip the penny and the other to represent the penny. The initial state for all Qubits
-is |0⟩(=|T⟩). To simulate Picard’s decision, we assume that he chooses randomly whether or not to flip the coin, in
-agreement with the optimal strategy for the classic penny-flip game. This random choice can be created by putting one
-qubit into an equal superposition, e.g. with the Hadamard gate H, and then measure its state. The measurement will show
-Heads or Tails with equal probability p=0.5.
-
-To simulate the penny flip game we take the second qubit and put it into its excited state |1⟩(=|H⟩) by applying the X
-(or NOT) gate. Q’s first move is to apply the Hadamard gate H. Picard’s decision about the flip is simulated as a CNOT
-operation where the control bit is the outcome of the random number generator described above. Finally Q applies a Hadamard
-gate again, before we measure the outcome.
-
-We first import the necessary tools
-
-.. code:: python
-
-    from pyquil import Program
-    from pyquil.api import WavefunctionSimulator
-    from pyquil.gates import *
-
-    wf_sim = WavefunctionSimulator()
-    prog = Program()
-    ro = prog.declare('ro', 'BIT', 2)
-
-Then we need to define two registers that will be used for the measurement of Picard’s decision bit and the final answer
-of the penny tossing game.
-
-.. code:: python
-
-    picard_register = ro[1]
-    answer_register = ro[0]
-
-We need to encode the two different actions of Picard, which conceptually is equivalent to an if-else control flow as:
-
-.. code:: python
-
-    then_branch = Program(X(0))
-    else_branch = Program(I(0))
-
-and then wire it all up into the overall measurement circuit:
-
-.. code:: python
-
-    prog.inst(X(0), H(1))
-    prog.inst(H(0))
-    prog.measure(1, picard_register)
-    prog.if_then(picard_register, then_branch, else_branch)
-    prog.inst(H(0))
-    prog.measure(0, answer_register)
-    print(prog)
-
-Finally we play the game several times
-
-.. code:: python
-
-    wf_sim.run_and_measure(prog, [0, 1], 10)
-
-Remember that the first number is the outcome of the game (value of the answer_register) whereas the second number is the
-outcome of Picard’s decision (value of the picard_register).
-
-No matter what Picard does, Q will always win!
-
-
+Multi-Qubit Basis Enumeration
+-----------------------------
+
+The Rigetti QVM enumerates bitstrings such that qubit `0` is the least significant bit (LSB)
```

Comment:
@mpharrigan did you imply this isn't the QVM, but only the wavefunction simulator, now? I ask because I wasn't otherwise planning on changing this section at all

Reply:
need 

```rst
``0``
```

(double backtick)

Reply:
QVM wavefunction simulations use qubit-zero-on-right which only matters if you're indexing into `wfn.amplitudes`. `QuantumComputer.run` has no baked-in assumption but by strong convention people do `MEASURE 0 ro[0]; MEASURE 1 ro[1]; ...` which results in a numpy array with qubit-zero-on-left. Therefore, you **will** get into trouble if you write a function that post-processes bitstrings and try to use it for sampled bitstrings (like from a real qpu) and on bitstrings whose probability you want to look up via `wfn.amplitudes`

cc @kmckiern @willzeng